pkg/controller/controller.go

```diff
@@ -504,7 +504,11 @@ func (c *Controller) syncHandler(key string) error {
 			}
 		}
 
-		time.Sleep(15 * time.Second)
+		time.Sleep(30 * time.Second)
```

Comment:
Without having this sleep, shall we query the pod status? We can check whether pod is ready. THen only we can proceed.

Reply:
Fixed